src/getHumanAge.test.js

```diff
@@ -7,4 +7,88 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should calculate cat/dog age correctly into human age', () => {
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test('should be set to zero for negative values', () => {
+    expect(getHumanAge(-1, -5)).toEqual([0, 0]);
+  });
+
+  test('should correctly convert first 15 cat years', () => {
```

Comment:
The test description 'should correctly convert first 15 cat years' is not accurate. It seems like it is expecting the first 15 cat years to be equivalent to 1 human year which is not clearly explained in the test description.